drivers/gpio/gpio_mcux.c

```diff
@@ -105,15 +105,20 @@ static int gpio_mcux_configure(struct device *dev,
 	 */
 
 	if (access_op == GPIO_ACCESS_BY_PIN) {
-		if ((flags & GPIO_INPUT) != 0) {
+		switch (flags & (GPIO_INPUT | GPIO_OUTPUT)) {
+		default:
```

Comment:
It's kind of weird to put the default first. Is there a reason you did that?

Reply:
Personal style, I think: it makes it immediately obvious that a default has been provided (you don't have to search for it at the end of the block, or look for it attached to a case label in the middle).  Looks like that idiom is also used in the bluetooth controller and some other Nordic stuff (though I didn't pick it up from them) and the littlefs wrapper (though that might be mine).

If it's bothersome I can move it to the bottom.

Reply:
Done

Reply:
thank you!